monero-wallet-generator.html

```diff
@@ -10938,14 +10933,23 @@ <h5 id="gpg-instructions">How to verify GPG signatures</h5>
 </footer>
 
 <script>
+
+if (document.getElementById("currencyDropDownList").value == "Monero") {
+  var cnUtil = cnUtilGen(moneroConfig);
```

Comment:
Does this not add a *local* variable ?

Reply:
It is global in JS (if it were in a function it would be local).
Code is not so readable though -- better to declare above then assign value inside the if / else (without the `var` keyword of course).